firefox-ios/Client/Extensions/UIView+ThemeUUIDIdentifiable.swift

```diff
@@ -6,7 +6,7 @@ import UIKit
 import Common
 import Shared
 
-extension UIView: ThemeUUIDIdentifiable {
+extension UIView: Common.ThemeUUIDIdentifiable {
```

Comment:
Just wondering, it is now mandatory from Xcode 16 to prefix the class with the package ?

Reply:
The short response is Yes for extension conformances that are part of a different module. For more details you can check the conversation above that I had with Matt Reagan. There I left more details about the error, etc.

Reply:
Ok perfect thank you @PARAIPAN9